neo.UnitTests/UT_Consensus.cs

```diff
@@ -93,7 +94,6 @@ public void ConsensusService_Primary_Sends_PrepareRequest_After_OnStart()
             Console.WriteLine($"header {header} hash {header.Hash} timstamp {timestampVal}");
 
             timestampVal.Should().Be(4244941696); //1968-06-01 00:00:00
-            TimeProvider.Current.UtcNow.ToTimestamp().Should().Be(4244941711); //1968-06-01 00:00:15
```

Comment:
this consumption was removed here too... but I can make it back if you prefer. We will re-write everything here as soon as we decide upon these fast tests.